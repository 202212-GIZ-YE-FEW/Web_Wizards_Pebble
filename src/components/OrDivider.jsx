const OrDivider = ({ text }) => {
    return (
        <div className='flex mt-7 order-3 items-center text-center mb-4'>
            <hr className=' text-gray-400  w-full ' />
            <label className='block font-medium text-sm text-gray-700 ml-6 mr-6'>
                {text}
            </label>
            <hr className=' text-gray-400 w-full' />
        </div>
    );
};
export default OrDivider;
