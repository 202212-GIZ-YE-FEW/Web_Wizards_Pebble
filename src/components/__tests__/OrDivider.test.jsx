import renderer from "react-test-renderer";

import OrDivider from "@/components/shared/OrDivider";

it("renders correctly", () => {
    const tree = renderer.create(<OrDivider />).toJSON();
    expect(tree).toMatchSnapshot();
});
