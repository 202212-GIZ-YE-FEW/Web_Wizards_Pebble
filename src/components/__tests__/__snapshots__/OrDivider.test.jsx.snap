// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="flex mt-7 order-3 items-center text-center mb-4"
>
  <hr
    className=" text-gray-400  w-full "
  />
  <label
    className="block font-medium text-sm text-gray-700 ml-6 mr-6"
  />
  <hr
    className=" text-gray-400 w-full"
  />
</div>
`;
